Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml

```diff
@@ -27,6 +27,7 @@
       - Prying
     useSound:
       path: /Audio/Items/crowbar.ogg
+  - type: ToolForcePowered
```

Comment:
They shouldn't be relying on anything Tool related.

Reply:
See above.